src/org/zaproxy/zap/extension/domxss/TestDomXSS.java

```diff
@@ -199,14 +210,13 @@ private WebDriver getNewFirefoxDriver() {
 	}
 
 	private WebDriverWrapper getFirefoxDriver() {
-		WebDriverWrapper fxDriver;
```

Comment:
This one should be kept (not the same driver as the instance), the method should also be changed to static (to emphasise that).